src/store/story.js

```diff
@@ -16,10 +16,16 @@ const makeFrameStyle = f => {
 
 export default {
   state: {
-    storyTitle: '', // @todo to be edited by user
     creationDate: null,
     currentFrameId: 1,
     frames: [
+      { // cover frame
```

Comment:
By adding the cover in the official list of frame, a lot of unrelated code is force to filter it out to still function properly. Would it be simpler to add it only in a getter used specifically by the viewer? The story title can be stored as a top-level field in the state.

Reply:
Initially I was planning to do that, therefore `storyTitle` existed, with this flow the `currentFrame` and `currentFrameId` will need to be updated according too.

I agree with the `filter( f => f.id === 0 )` is really unrelated and not clear, I have one idea to try out, let me do the update later here.

Reply:
Done, please test this carefully once again.